contrib/container-images/centos-8-pypy.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
+# Builds an ARA API server container image using the latest PyPi packages on CentOS 8.
+build=$(buildah from centos:8)
+
+# Get all updates, install pip, database backends and gunicorn application server
+# This lets users swap easily from the sqlite default to mysql or postgresql just by tweaking settings.yaml.
+# Note: We use the packaged versions of psycopg2 and mysql python libraries so
+#       we don't need to install development libraries before installing them from PyPi.
```

Comment:
nit: this statement is half accurate after changing from pymysql to mysqlclient

Reply:
Renamed through last commit